### PR TITLE
Changes logging for base worker

### PR DESCRIPTION
**Pull request by dharmit**

### CODE REVIEW

File: container_pipeline/workers/base.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def notify(self, data):
 
     def export_logs(self, logs, destination):
         """"Write logs in given destination"""
-        self.logger.debug('Writing build logs to NFS share')
+        self.logger.info('Writing build logs to NFS share')
```
Comment (dharmit) [inline container_pipeline/workers/base.py:34]: Changed this to `info` as this seems more like an informative message we're storing in the logs.

File: container_pipeline/workers/base.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ def export_logs(self, logs, destination):
         except IOError as e:
             self.logger.critical("Failed writing logs to {}"
                                  .format(destination))
-            self.logger.critical(str(e))
+            self.logger.error(str(e))
```
Comment (dharmit) [inline container_pipeline/workers/base.py:45]: Changed this to `error` as we're logging the error `e`.

**Review by navidshaikh — APPROVED** [review]
LGTM

**Comment by rtnpro** [comment]
Megred these changes to manually.

**Pull request closed without merging**